src/lib/constants.ts

```diff
@@ -99,3 +99,5 @@ export const POST_IMG_MAX = {
   height: 2000,
   size: 1000000,
 }
+
+export const LINK_META_PROXY = 'https://cardyb.staging.bsky.dev/v1/extract?url='
```

Comment:
I actually deployed the production stack of the cardyb service now, so we can use this URL for production. The staging URL also works (but should be used for staging, ideally, although it's fine if they both use production)

```
export const LINK_META_PROXY = 'https://cardyb.bsky.app/v1/extract?url='
```

Reply:
Alright lemme fix that